extensions/ql-vscode/src/run-queries.ts

```diff
@@ -345,7 +345,7 @@ async function promptUserToSaveChanges(document: TextDocument): Promise<boolean>
     else {
       const yesItem = { title: 'Yes', isCloseAffordance: false };
       const alwaysItem = { title: 'Always Save', isCloseAffordance: false };
-      const noItem = { title: 'No (run anyway)', isCloseAffordance: false };
+      const noItem = { title: 'No (run version on disk)', isCloseAffordance: false };
```

Comment:
Does 'the last saved version on disk' make sense?

Reply:
When would it ever _not_ be the last saved version (unless you have a funky filesystem that automatically saves backup versions)?

Reply:
I expect never: what you've written is correct, I'm just wondering whether we can make the message even more obvious.